chain-signatures/node/src/protocol/signature.rs

```diff
@@ -199,8 +202,9 @@ impl SignatureGenerator {
 
     pub fn poke(&mut self) -> Result<Action<FullSignature<Secp256k1>>, ProtocolError> {
         if self.sign_request_timestamp.elapsed() > self.timeout_total {
+            tracing::warn!("signature protocol timed out completely");
             return Err(ProtocolError::Other(
-                anyhow::anyhow!("signature protocol timeout completely").into(),
+                anyhow::anyhow!("signature protocol timed out completely").into(),
```

Comment:
I wouldn't rename it `timed out` because it's much easier to search for `timeout` even though `timed out` is more grammatically correct